doc/installing.rst

```diff
@@ -111,9 +111,9 @@ dependencies:
   numpy >=1.17.
 - **all other libraries:** 12 months
 
-The above should be interpreted as *the minor version (X.Y) initially published no more
-than N months ago*. Patch versions (x.y.Z) are not pinned, and only the latest available
-at the moment of publishing the xarray release is guaranteed to work.
+The above should be interpreted as *the most recent minor version (X.Y) initially
+published more than N months ago*. Patch versions (x.y.Z) are not pinned, and only the
```

Comment:
How about this? (from [here](https://github.com/numpy/numpy/pull/14086#issuecomment-649762395))

```suggestion
This means the latest minor (X.Y) version from N months prior.
Patch versions (x.y.Z) are not pinned, and only the
```

Reply:
The suggested change is very clear to me.

Reply:
if anyone feels a particular wording is much easier to understand feel free to push to this branch